src/samples/ExtendedSlashScreen.Uno.Samples/ExtendedSlashScreen.Uno.Samples.Mob/ExtendedSlashScreen.Uno.Samples.Mobile.csproj

```diff
@@ -0,0 +1,89 @@
+	</PropertyGroup>
+	<ItemGroup>
+	  <None Remove="Android\Resources\drawable-xhdpi\splashscreen.png" />
+	  <None Remove="Android\Resources\drawable\Icon.png" />
+	  <None Remove="Android\Resources\drawable\splash_screen.xml" />
+	</ItemGroup>
+	<ItemGroup>
+		<PackageReference Include="Uno.WinUI" Version="4.6.18" />
+		<PackageReference Include="Uno.WinUI.RemoteControl" Version="4.6.18" Condition="'$(Configuration)'=='Debug'" />
+		<PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.6.18" />
+		<PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.1.0" />
+		<PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.0" />
+	</ItemGroup>
+	<ItemGroup>
+		<ProjectReference Include="..\..\..\library\ExtendedSplashScreen.Uno.WinUI\ExtendedSplashScreen.Uno.WinUI.csproj" />
+	</ItemGroup>
+	<Choose>
+		<When Condition="'$(TargetFramework)'=='net6.0-android'">
+			<ItemGroup>
+				<PackageReference Include="Xamarin.Google.Android.Material" Version="1.4.0.4" />
+				<PackageReference Include="Uno.UniversalImageLoader" Version="1.9.36" />
+			</ItemGroup>
+			<ItemGroup>
+				<AndroidEnvironment Include="Android/environment.conf" />
+			</ItemGroup>
+		</When>
+		<When Condition="'$(TargetFramework)'=='net6.0-ios'">
+			<PropertyGroup Condition="'$(TargetFramework)'=='net6.0-ios'">
+				<MtouchExtraArgs>$(MtouchExtraArgs) --setenv=MONO_GC_PARAMS=soft-heap-limit=512m,nursery-size=64m,evacuation-threshold=66,major=marksweep,concurrent-sweep</MtouchExtraArgs>
+				<!-- See https://github.com/unoplatform/uno/issues/9430 for more details. -->
+				<MtouchExtraArgs>$(MtouchExtraArgs) --registrar:static</MtouchExtraArgs>
+				<!-- https://github.com/xamarin/xamarin-macios/issues/14812 -->
+				<MtouchExtraArgs>$(MtouchExtraArgs) --marshal-objectivec-exceptions:disable</MtouchExtraArgs>
+			</PropertyGroup>
+			<ItemGroup>
+				<PackageReference Include="Uno.Extensions.Logging.OSLog" Version="1.4.0" />
+			</ItemGroup>
+		</When>
+		<When Condition="'$(TargetFramework)'=='net6.0-maccatalyst'">
+			<PropertyGroup>
+				<!-- Configure the GC -->
+				<MtouchExtraArgs>$(MtouchExtraArgs) --setenv=MONO_GC_PARAMS=soft-heap-limit=512m,nursery-size=64m,evacuation-threshold=66,major=marksweep,concurrent-sweep</MtouchExtraArgs>
+				<!-- Required for unknown crash as of .NET 6 Mobile Preview 13 -->
+				<MtouchExtraArgs>$(MtouchExtraArgs) --registrar:static</MtouchExtraArgs>
+				<!-- https://github.com/xamarin/xamarin-macios/issues/14812 -->
+				<MtouchExtraArgs>$(MtouchExtraArgs) --marshal-objectivec-exceptions:disable</MtouchExtraArgs>
+				<!-- Full globalization is required for Uno -->
+				<InvariantGlobalization>false</InvariantGlobalization>
+			</PropertyGroup>
+			<ItemGroup>
+				<PackageReference Include="Uno.Extensions.Logging.OSLog" Version="1.4.0" />
+			</ItemGroup>
+		</When>
+		<When Condition="'$(TargetFramework)'=='net6.0-macos'">
```

Comment:
Is this necessary since there's nothing inside?

Reply:
I guess I will remove it.